note.py

```diff
@@ -378,6 +378,8 @@ def __init__(self, master, model, icons):
         self.pack()
         self.__create_widgets(icons)
         model.on_selection_changed.subscribe(self.update)
+        first_note = list(self.model.notes.keys())[0]
```

Comment:
This fails with an exception, when no note is present. The following suggestion should solve the issue:

```suggestion
        first_note = list(self.model.notes.keys())[0] if len(self.model.notes) > 0 else None
```